### PR TITLE
Bump lib versions 

### PR DESCRIPTION
**Pull request by egs33**
### Changelogs
- https://github.com/clojure/clojure/blob/clojure-1.11.1/changes.md
- https://github.com/clj-commons/camel-snake-kebab/blob/master/NEWS.md
- https://github.com/cognitect-labs/aws-api/blob/v0.8.652/CHANGES.md
- https://github.com/liquidz/build.edn/blob/0.9.203/CHANGELOG.adoc
- https://github.com/lambdaisland/kaocha/blob/v1.80.1274/CHANGELOG.md
- https://github.com/clojure/core.async#changelog
- https://github.com/clojure/tools.logging/blob/v1.2.4/CHANGELOG.md
- https://github.com/pyr/unilog#releases

### CODE REVIEW

File: src/gluttony/sqs.clj [hunk 1]
```diff
@@ -4,13 +4,13 @@
    [camel-snake-kebab.core :as csk]
    [camel-snake-kebab.extras :as cske]
    [clojure.core.async :as a]
-   [cognitect.aws.client.api.async :as aws-async]))
```
Comment (egs33) [inline src/gluttony/sqs.clj:7]: `cognitect.aws.client.api.async` is deprecated 

https://github.com/cognitect-labs/aws-api/blob/a532fe1c14794dd8e4f9fd55750d4519ad218b8a/src/cognitect/aws/client/api/async.clj#L4-L10

File: src/gluttony/record/consumer.clj [hunk 2]
```diff
@@ -9,7 +9,7 @@
    [gluttony.record.message :as r.msg]
    [gluttony.sqs :as sqs])
   (:import
-   (cognitect.aws.client
```
Comment (egs33) [inline src/gluttony/record/consumer.clj:12]: This ns is moved at https://github.com/cognitect-labs/aws-api/commit/e262aac7a5cd2c1214a632ba851f95b6686a9b0f#diff-be06d82173b4b88bcf4f60c30239fb9ede8dd25bf113da07259435b019069652R4

**Review by liquidz — APPROVED** [review]
LGTM. Thank you!

**Comment by egs33** [comment]
:pray:

**Pull request merged**